README.md

```diff
@@ -0,0 +1,25 @@
+# Clearbit Go Bindings
+
+
+## Usage
+
+
+```go
+package main
+
+import (
+  "fmt"
+  "github.com/clearbit/clearbit-go/clearbit"
+  "os"
+)
+
+func main() {
+  client := clearbit.NewClient(nil, os.Getenv("CLEARBIT_KEY"))
```

Comment:
Feels a little odd having to pass in Nil here. Could make a config struct that has sensible defaults?